src/confd/yang/infix-hardware@2024-01-18.yang

```diff
@@ -25,6 +25,66 @@ module infix-hardware {
     description "A two-letter country code.";
   }
 
+  identity hardware-class {
+    description "infix hardware base class";
+  }
+
+  identity usb {
+    base hardware-class;
+    description "This identity is used to describe a USB port";
+  }
+  identity cpu {
+    base hardware-class;
+    description "This identity is used to describe a USB port";
+  }
+
+  identity power {
+    base hardware-class;
+    description "This identity is used to describe a USB port";
+  }
+
+  identity product {
+    base hardware-class;
+    description "This identity is used to describe a USB port";
+  }
```

Comment:
Copypasta detected :robot: 

Even so...do we want to represent these as 3 distinct classes, or are they all of class `vpd`?